azure-ts-vm-provisioners/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: aws-ts-ec2-provisioners
```

Comment:
maybe give this example one more set of review to make sure it's Azure and not AWS  (several places ... )